test/uk/gov/hmrc/cipphonenumberverification/controllers/OtpControllerSpec.scala

```diff
@@ -16,7 +16,7 @@
 
 package uk.gov.hmrc.cipphonenumberverification.controllers
 
-import org.mockito.ArgumentMatchersSugar.any
+import org.mockito.ArgumentMatchers.any
```

Comment:
We should stick to using ArgumentMatchersSugar

Reply:
when I use the sugar one, I get other problems.  All the other tests have matchers. (as opposed to the sugar one)

Reply:
all the other unit tests use Matchers

Reply:
all the other tests use ArgumentMatchersSugar.

Reply:
they all say Matchers